test/nightwatch/add-voucher-to-cart.js

```diff
@@ -46,8 +46,9 @@ module.exports = {
       .click('.btn-add-to-cart')
       .pause(pauseOnClick)
       .assert.urlContains('/basket')
-      .assert.containsText('.line-item:nth-of-type(1) .product-description p:nth-of-type(1)', voucher.name)
-      .assert.containsText('.line-item:nth-of-type(1) .product-description p:nth-of-type(2)', 'Full Day')
+      .pause(250)
```

Comment:
Presumably because the basket contents come from the API and are put on the page dynamically? If you can `waitForElementVisible()` will usually allow you to achieve the same effect, and indicates to the reader that some async stuff is going on. Use `pause()` only as a last resort!